python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -360,8 +359,13 @@ def _build_func_common(measure_input, check_gpu=None, cuda_arch=None, build_opti
         if cuda_arch:
             set_cuda_target_arch(cuda_arch)
 
-        with build_config(**opts):
-            func = build(s, args, target_host=task.target_host)
+        if measure_input.target.device_name == 'vta':
```

Comment:
It is better to create vta's own function rather than insert an "if" to the common part

Reply:
Originally that was the decision, but I was suggested by Jared and Tianqi to unify which led to adding the check in here.

Reply:
I've guarded the if against potential errors (say if you initialize with `target = 'llvm'` instead of `target=tvm.target.arm_cpu()`). Let me know if you're okay to proceed this way. I think we'll need to find a longer term solution to building the module for VTA without needing the special  `vta.build()`